app/Models/EntryDAO.php

```diff
@@ -1197,10 +1197,15 @@ public function listByIds(array $ids, string $order = 'DESC') {
 
 	/**
 	 * For API
-	 * @return array<string>
+	 * @return array<string>|false
 	 */
-	public function listIdsWhere($type = 'a', $id = '', $state = FreshRSS_Entry::STATE_ALL,
-			$order = 'DESC', $limit = 1, $firstId = '', $filters = null): array {
+	public function listIdsWhere($type = 'a',
```

Comment:
Why no types here?

Reply:
your right ! I have forgotten

Reply:
i remeber... I focused on Factory.php this file does not need to be fully modified for phpstan to pass. my target is factory.php

Reply:
Yes fine, but then please avoid reformatting

Reply:
done